docs/prometheus.md

```diff
@@ -0,0 +1,330 @@
+kubectl apply --server-side --force-conflicts -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.59.2/example/prometheus-operator-crd/monitoring.coreos.com_probes.yaml
+kubectl apply --server-side --force-conflicts -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.59.2/example/prometheus-operator-crd/monitoring.coreos.com_prometheuses.yaml
+kubectl apply --server-side --force-conflicts -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.59.2/example/prometheus-operator-crd/monitoring.coreos.com_prometheusrules.yaml
+kubectl apply --server-side --force-conflicts -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.59.2/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
+kubectl apply --server-side --force-conflicts -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.59.2/example/prometheus-operator-crd/monitoring.coreos.com_thanosrulers.yaml
+```
+
+Unfortunately there is no easy way to check which Custom Resource Definition is applied on the cluster.
+The only symptom is failing installation with the following or similar error (containing `com.coreos.monitoring.v1`).
+
+```text
+Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(Prometheus.spec): unknown field "hostNetwork" in com.coreos.monitoring.v1.Prometheus.spec
+```
+
+Required Custom Resource Definitions are listed in [Kube Prometheus Stack repository][kube-prometheus-stack-repo]
+
+[kube-prometheus-stack-repo]: https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack#upgrading-an-existing-release-to-a-new-major-version
+
+### Installing Sumo Logic Prometheus Operator side by side with existing Operator
+
+You can have multiple Prometheus Operators in the cluster, but you need to ensure that they do not observe the same namespace.
+__Every__ Operator should have `${NAMESPACE}` as part of `--deny-namespaces` argument or shouldn't have it as part of `--namespaces` argument,
+for example:
+
+```shell
+$ kubectl get deployment -l app=kube-prometheus-stack-operator -oyaml -A
+apiVersion: v1
+items:
+- apiVersion: apps/v1
+  kind: Deployment
+...
+      spec:
+        containers:
+        - args:
+...
+          - --deny-namespaces=${NAMESPACE},some-namespace
+...
+- apiVersion: apps/v1
+  kind: Deployment
+...
+      spec:
+        containers:
+        - args:
+...
+          - --namespaces=some-namespace
+...
+```
+
+#### Set Sumo Logic Prometheus Operator to observe installation namespace
+
+Limit observed namespaces to `${NAMESPACE}` in `user-values.yaml`
+
+```yaml
+kube-prometheus-stack:
+  prometheusOperator:
+    namespaces:
+      releaseNamespace: true
+```
```

Comment:
Imho it should be default for our collection

Reply:
rethinked it, and defaulting it can be problematic for setups where we are the only Proemtheus in the cluster